hugo/content/info/corrections.md

```diff
@@ -69,3 +70,12 @@ Here are some good examples:
 Following these instructions will help us to process corrections more quickly.
 We will do our best to process revisions or errata within four weeks, but make no guarantees as to the processing time.
 Submissions not meeting these standards will be rejected, potentially without notice.
+
+Changes to the author list are potentially subject to further review.
+
+### Note on author changes
+
+Some conferences and workshops have a policy of disallowing authorship changes after the initial submission.
+Because it is beyond our ability to track this, changes to the list of authors, whether in the PDF or in the metadata, require permission from the associated conference program chair or workshop/tutorial chair.
+You are responsible for arranging for this permission to be conveyed to us, either via email or on your Github issue.
+We will make exceptions for the correction of obvious spelling errors or other typos, and for author name changes (i.e., the list of authors is unchanged, but one of the authors changes their name).
```

Comment:
Instead of "We will make exceptions for ...", maybe write
```
Obvious spelling errors or other typos, and author name changes (i.e., the list of authors is unchanged, but one of the authors changes their name) do not need permission.
```
With the current wording, it is not completely clear what the exceptions refer to (i.e. what is the effect of an exception?)